### DISCUSSION TITLE
Server keeps crashing

### DISCUSSION THREAD

**Discussion by cbytestech** [itzg/docker-minecraft-server#1935] [Q&A]
Running Bedrock Server with script listed. 
My server keeps crashing, Error on PS4 is disconnected, but when I ssh into server @ liquidweb, I have to restart the container.
My ISP, xfin, does have issues here and there as far as connectivity, but at those issues I just disable, re enable the wifi on phone and stop/start mineproxydroid.
I have read about memory limit not sure if the issue is that it is a memory leak or its reaching max. Have read to limit to 2GB. 
1. Is there a better way to connect to server IP with PS4 than this app?
2. Is this memory leak issue accurate?
3. Also, what command would I use to do that without overwriting my current world files

containername mctestserver
have checked status and when connecting it fluctuates as far as performance(Image attached)
![Screenshot_20230127-014537_Mobile SSH - Premium](https://user-images.githubusercontent.com/24298010/215199286-74a7ca9d-ffc0-434a-82cb-1728050925ac.jpg)

**Comment by itzg** [comment]
I'm not sure if there's anything I can do from the image. I have heard that bedrock server has memory leak issues in general and people have to do schedules restarts.

I'm going to convert this to a discussion and I recommend you ask around in the Discord server.
